pkg/controller/installation/products/solutionexplorer/reconciler.go

```diff
@@ -164,10 +165,11 @@ func (r *Reconciler) ReconcileCustomResource(ctx context.Context, inst *v1alpha1
 		Template: webapp.WebAppTemplate{
 			Path: defaultTemplateLoc,
 			Parameters: map[string]string{
-				param_oauth_client:         defaultSubNameAndPkg,
-				param_sso_route:            ssoConfig.GetHost(),
-				param_openshift_host:       inst.Spec.MasterURL,
-				param_openshift_oauth_host: oauthURL,
+				paramOauthClient:        defaultSubNameAndPkg,
+				paramSSORoute:           ssoConfig.GetHost(),
+				paramOpenShiftHost:      inst.Spec.MasterURL,
+				paramOpenShiftOauthHost: oauthURL,
+				paramOpenShiftVersion:   "4",
```

Comment:
Can we use `controllerutil.CreateOrUpdate` to update Cr spec to ensure upgrades are handled correctly